doc/man3/d2i_PrivateKey.pod

```diff
@@ -86,6 +86,9 @@ functions described in L<d2i_X509(3)>.
 
 =head1 NOTES
 
+All the functions that work with data in memory update the data pointers I<*pp>
```

Comment:
All the functions that operate on data in memory update the data pointer I<*pp> after a successful operation, just like the other d2i and i2d functions; see L<d2i_X509(3)>.

Reply:
Fixup pushed.